arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -566,153 +572,2046 @@
+          }
+        },
+        "TopicPath": {
+          "type": "string",
+          "description": "Path of the topic that this subscription description belongs to."
+        },
+        "UpdatedAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The exact time the message has been updated."
+        },
+        "UserMetadata": {
+          "type": "string",
+          "description": "The user metadata."
+        }
+      },
+	  "x-ms-client-flatten": true,
+      "description": "Represents the metadata description of the Subscription."
+    },
+    "SubscriptionResource": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "URL of the resource."
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the subscription."
+        },
+        "type": {
+          "type": "string",
+          "description": "The name of the resource."
+        },
+        "location": {
+          "type": "string",
+          "description": "Supported Azure location in which the resource exists."
+        },
+        "properties": {
+          "$ref": "#/definitions/SubscriptionProperties",
+          "description": "Properties of the Subscription."
+        }
+      },
+      "description": "Description of Subscription Resource."
+    },
+    "SubscriptionListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SubscriptionResource"
+          },
+          "description": "Result of the List Subscriptions operation."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Link to the next set of results. Not empty if Value contains incomplete list of Subscriptions"
+        }
+      },
+      "description": "The response of the List Subscriptions operation."
+    },
+    "TopicCreateOrUpdateParameters": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Topic name."
+        },
+        "type": {
+          "type": "string",
+          "description": "ARM type of the namespace."
+        },
+        "location": {
+          "type": "string",
+          "description": "Location of the resource."
+        },
+        "properties": {
+          "$ref": "#/definitions/TopicProperties",
+          "description": "Properties of the Topics."
+        }
+      },
+      "required": [
+        "location"
+      ],
+      "description": "Parameters supplied to the CreateOrUpdate Topic operation."
+    },
+    "TopicProperties": {
+      "properties": {
+        "AccessedAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Message was sent, or the last time there was a receive request to this topic."
+        },
+        "Authorization": {
+          "type": "object",
+          "description": "AuthorizationRules."
```

Comment:
type object ??
How should this be [de]serialized? Is this a dictionary or an actual model type?